_projects/2023/100430897/100430897.Rmd

```diff
@@ -0,0 +1,1048 @@
+---
+title: "Trust in government by race and ethnicity"
+description: |
+  Among Asian, Hispanic and Black adults, 23%, 23% and 21% respectively say they trust the federal government “most of the time” or “just about always” – higher levels of trust than among White adults (13%). During the last Democratic administration, Black and Hispanic adults similarly expressed more trust in government than White adults. Throughout most recent Republican administrations, White Americans were substantially more likely than Black Americans to express trust in the federal government to do the right thing.
```

Comment:
Maybe this belons to the introduction, and it's better to put here something more descriptive of your work here.